tests/unit/test_utils.py

```diff
@@ -1,7 +1,25 @@
 """Unit tests for utils.py"""
+import builtins
 from concurrent import futures
 
-from s3_tools.utils import _get_future_output
+import pytest
+
+from s3_tools.utils import (
+    _create_progress_bar,
+    _get_future_output
+)
+
+
+@pytest.fixture
+def hide_available_pkg(monkeypatch):
+    import_orig = builtins.__import__
+
+    def mocked_import(name, *args, **kwargs):
+        if "rich." in name:
```

Comment:
```suggestion
        if "rich" in name:
```

Reply:
The imports from rich package are partial, so they appear as `rich.progress`.

Reply:
Ok :smiley: